.github/workflows/run-k8s-tests-on-aks.yaml

```
@@ -46,11 +46,16 @@ jobs:
             -p "${{ secrets.AZ_PASSWORD }}" \
             --tenant "${{ secrets.AZ_TENANT_ID }}"
 
+      - name: Format cluster name
+        run: |
+          rev=$(echo "${{ github.event.pull_request.head.sha }}" | cut -c1-12)
```

```suggestion
          rev=$(git rev-parse --short HEAD)
```

We can do that as at this point the repo has been pulled and we're at the `github.event.pull_request.head.sha` commit.